### DISCUSSION TITLE
Problem during the installation of the clabernetes-manager

### DISCUSSION THREAD

**Discussion by rogerscuall** [srl-labs/containerlab#2243] [Q&A]
I'm following the step by step guide provided here:
https://containerlab.dev/manual/clabernetes/quickstart/#installing-clabernetes
This used to be working for me, but it is not longer working.
After running the: 
```
helm upgrade --install --create-namespace --namespace c9s \
clabernetes oci://ghcr.io/srl-labs/clabernetes/clabernetes
```
I suppose to see the 3 pods running the manager, but they never get ready and they keep crashing.
The logs of all 3 pods show this:
```
Defaulted container "manager" out of: manager, init (init)
Error from server (BadRequest): container "manager" in pod "clabernetes-manager-549b5b7597-v27jv" is waiting to start: PodInitializing
```
This was working previously for me, so I moved back a couple of versions and still got something similar.

**Comment by hellt** [comment]
at this point it is a cluster issue, you will have to check whats going on there
